src/components/general/Navbar/Notifica/Notifications/index.js

```diff
@@ -105,10 +107,12 @@ const requestOptions = {
             if (dateRange === '30d') {
                 lastDate.setDate(lastDate.getDate() - 30);
             }
-            return {
-                isPending: !!isPending,
+            const a = {
+                is_pending: !!isPending,
                 timestamp__gte: getDateWithTimezone(encodeDate(lastDate)),
             };
+            console.warn('url params', a);
+            return a;
```

Comment:
The variable name should be descriptive.
Let's not leave console.warn meant for debugging.